### ISSUE TITLE
Accessibility - GIFs

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#85]
### Improving QDR's Dataverse for Qualitative Data: https://qdr.syr.edu/qdr-blog/improving-qdrs-dataverse-qualitative-data

Resources: https://www.w3.org/TR/WCAG20-TECHS/G152.html ; https://accessibility.psu.edu/animations/
(5 Second Rule - GIFs on webpages should be able to be stopped or stop after 5 seconds automatically; accessibility preference is to avoid GIFs).

**Comment by qqmyers** [comment]
This looks like  a content issue where the GIFs should be altered/replaced (versus something to change in code/css/etc.) Should it be assigned to someone other than me?

**Comment by kaitlinnewson** [comment]
In PKP documentation, we resolved this by converting GIFs to MP4 videos, and embedding them with `<video>` elements. This is one way to get the play/pause functionality that is necessary for this kind of content.

**Comment by adam3smith** [comment]
I've removed the gifs for now -- they don't add that much & the video is right there.
